nym-validator-rewarder/src/config/mod.rs

```diff
@@ -249,6 +251,9 @@ pub struct NyxdScraper {
     // if the value is missing, use `nothing` pruning as this was the past behaviour
     #[serde(default = "PruningOptions::nothing")]
     pub pruning: PruningOptions,
+
+    /// Specifies whether to store pre-commits within the database.
+    pub store_precommits: bool,
```

Comment:
you need to add `#[serde(default = true)]` so that old configs wouldn't fail to get deserialised

Reply:
Turns out it doesn't allow a `bool` to be default so had to hack around it